### PR TITLE
improve score error handling on proposal page

### PR DESCRIPTION
**Pull request by mktcode**
Changes proposed in this pull request:
- adjusted error message
- retry button
- comment header in component

The retry button is shown for everyone, since the score api handles the potential spam nicely.
The error message still mentions the misconfigured strategy issue because that's still possible and people should still check that first when they are unsure.

https://user-images.githubusercontent.com/6792578/155226672-9d6deecb-55c6-4c87-848a-85dc35c0e6ca.mp4

(has sound)

### CODE REVIEW

**Review by samuveth — COMMENTED** [review]
Will this "retry" also trigger recalculation on the score endpoint? We could consider adding that trigger to the frontend too, then anyone can trigger it needs to be recalculated

**Comment by mktcode** [comment]
It covered both. The video should show everything.

**Comment by samuveth** [comment]
> It covers both. The video should show everything. Or what do you mean?

Oh the video has sound 🤣 Amazing!

File: src/components/Block/ResultsError.vue [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
+}
 </script>
 
 <template>
   <Block :title="$t('results')" icon="warning" iconClass="text-red">
     <div>{{ t('resultsError') }}</div>
+    <UiButton class="mt-3 w-full" @click="retry" :loading="retrying">
+      <Icon name="refresh" />
+      {{ t('retry') }}
+    </UiButton>
```
Comment (samuveth) [inline src/components/Block/ResultsError.vue:42]: This button could be primary
```suggestion
    <UiButton class="mt-3 w-full" @click="retry" :loading="retrying">
      <Icon name="refresh" />
      {{ t('retry') }}
      primary
    </UiButton>
```

**Review by samuveth — APPROVED** [review]
👍

File: src/components/Block/ResultsError.vue (hunk 1 above)
Comment (mktcode) [reply src/components/Block/ResultsError.vue:42]: Hmmmm idk... isn't that more for forms and saving something and so on... well.... actually.. why not. :D

**Pull request merged**